frontend/src/app/settings/core/announcement-settings/edit-announcement/edit-announcement/edit-announcement.component.html

```diff
@@ -0,0 +1,66 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<form [formGroup]="announcementForm" class="collab-card space-y-2">
+  <mat-form-field class="w-full" appearance="fill" subscriptSizing="dynamic">
+    <mat-label>Title</mat-label>
+    <input matInput formControlName="title" />
+  </mat-form-field>
+  <mat-form-field class="w-full" appearance="fill" subscriptSizing="dynamic">
+    <mat-label>Description</mat-label>
+    <textarea matInput formControlName="message"></textarea>
+    @if (
+      announcementForm.get("message")?.getError("titleOrDescriptionAvailable")
+    ) {
+      <mat-error>Please enter a description or title!</mat-error>
+    }
+  </mat-form-field>
+
+  <div class="flex items-center gap-2">
```

Comment:
```suggestion
  <div class="flex items-start gap-2">
```

Otherwise the Dismissible checkbox is no longer aligned when the mat-error is displayed (`items-start` fixes it in my tests): 
![image](https://github.com/user-attachments/assets/3f68e1b3-15d2-4083-afb4-aa68ac3733c7)